plugins/mtd-recovery/README.md

```diff
@@ -0,0 +1,63 @@
+# MTD Recovery
+
+This plugin allows a system without an exported MTD device to be updated by matching the DMI
+details of the platform. This allows updating SPI devices that are hidden behind a MUX which is
+very common when running fwupd on a BMC.
+
+This plugin supports the following protocol ID:
+
+* org.infradead.mtd
+
+## GUID Generation
+
+These devices are generated from the baseboard vendor ID:
+
+* `MTD\VEN_Lenovo&DEV_ThinkPad`
+* `MTD\VEN_Lenovo`
```

Comment:
If the most common case is from behind a BMC then I think this is very unlikely to ever show up in a laptop...  Maybe a different example from a server makes sense.

Reply:
Yup, changed.